dataactbroker/helpers/pandas_helper.py

```diff
@@ -33,11 +36,16 @@ def check_dataframe_diff(new_data, model, id_col, sort_cols, lambda_funcs=None):
 
     sess = GlobalDB.db().session
     current_data = pd.read_sql_table(model.__table__.name, sess.connection(), coerce_float=False)
+
+    # Apply any lambda functions provided to update values if needed
+    for col_name, lambda_func in lambda_funcs:
+        current_data[col_name] = current_data.apply(lambda_func, axis=1)
+
     # Drop the created_at and updated_at for the same reason as above, also drop the pk ID column for this table
     try:
-        current_data.drop([id_col, 'created_at', 'updated_at'], axis=1, inplace=True)
+        current_data.drop([id_col, 'created_at', 'updated_at'] + fk_cols, axis=1, inplace=True)
     except ValueError:
-        logger.info('id column, created_at, or updated_at not found, drop skipped.')
+        logger.info('id column, fk_columns, created_at, or updated_at not found, drop skipped.')
```

Comment:
Based on the above comment, this log should probably be more of `created_at, updated_at, or at least one of the columns provided for deletion not found, drop skipped.`